ui-frontend/packages/catalog-ui-search/src/main/webapp/component/query-settings/source-selector.tsx

```diff
@@ -87,27 +87,13 @@ const shouldBeSelected = ({
  * If it includes 'remote', that that means everything remote.  All other values are singular selections of a source.
```

Comment:
Does this type need to be updated to remove 'enterprise'? https://github.com/codice/ddf-ui/blob/7fbc7060ccceb0f98184eb59942a742a999c8bab/ui-frontend/packages/catalog-ui-search/src/main/webapp/component/query-settings/source-selector.tsx#L21

Reply:
Sort of, it should be 'all' 'fast' 'slow' string now